src/components/ReceiverInput.tsx

```diff
@@ -82,7 +82,8 @@ function ReceiverInput({ setError }: Props) {
     dispatchTransaction(TransactionActionCreators.setGenericAccount(null));
```

Comment:
This group of dispatchers will fly away in PR: https://github.com/paritytech/parity-bridges-ui/pull/189/files#diff-c41ea0157ecc87ff655874eaec216c1c80b01f0546bcb6d66cd44565aeae6a0cR71